docs/topics/modules/e2e/analyze_che_dependencies.adoc

```diff
@@ -0,0 +1,30 @@
+[#analyze_che_dependencies]
+= Analyzing Che Dependencies
+
+The Che workspace automatically analyzes dependencies for the code. To see how this analyzer works:
+
+. Return to the Che workspace browser tab.
+. In the `Project Explorer` panel, double click the `pom.xml` file.
++
+image::pomxml.png[Pom.xml File]
++
+. In the file, go to line 43 and add a line break.
+. Add the following XML code to the file after the line break:
++
+[source,xml]
+----
+<dependency>
+  <groupId>ch.qos.logback</groupId>
+  <artifactId>logback-core</artifactId>
+  <version>1.1.0</version>
+</dependency>
+----
++
+. An error icon appears near the line numbers when you add the XML code. Mouse over the icon to view the error message (see screen capture).
```

Comment:
'Mouse over' as a verb is too jargony. The IBM Style Guide suggests 'point to' or 'move the mouse pointer over'.